oeps/architectural-decisions/oep-0058-arch-translations-management.rst

```diff
@@ -0,0 +1,210 @@
+similar strings to be automatically translated, from the old projects to this new one. By
+moving all the Translation Files to the same repository we can increase the reach of the
+Transifex Memory feature to help translate similar strings across the entire
+code/documentation base.
+
+Impacts
+*******
+
+Impact on Translators
+=====================
+
+As we approach the end of the translation upgrade process, we will need to tactically
+move from multiple transifex projects to a single project.  This will require
+coordination with our translators to ensure that moving forward they are providing
+translations in the right place.
+
+Impact on Site Operators
+========================
+
+Currently the translation files for any given service or library is stored at the same
+place as the code, this has generally simplified the deployment story in the past.  With
+this change, the translations files will move to their own repository.  As we deprecate
+the old translations files, the relevant deployment tooling will need to be updated to
+pull down the translations from the new repository as a part of the deployment process.
+This will impact both the old Ansible based tooling as well as any new docker based
+tooling.
+
+Impact on Developers
+====================
+
+While it won’t directly impact the day-to-day workflow of developers, due to the same
+reasons that we impact site operators(new translations location), we will have to update
+development tools as well.
+
+Locations
+*********
+
+Dumps of the translation/localization files from Transifex for the Open edX Release
+project already exist in a repository with the name of openedx/openedx-i18n. A new
+repository named openedx/openedx-translations, will be similarly structured, but contain
+the translation files for all repositories within openedx. The GitHub Transifex app will
+be installed in the openedx organization. Similar to how the Build-Test-Release Working
+Group creates a new branch for each new named release of edx-platform, translation
+releases can also be kept in branches corresponding to edx-platform releases.
+
+Rejected Alternatives
+*********************
+
+Rewriting the Current Tooling for the New API
+=============================================
+
+The source code for the edx-transifex-bot is missing. We could rewrite the current
+tooling to try to solve the problems encountered in the last two Open edX releases and
+upgrade to the new API, but this approach would require a full rewrite, potentially more
+expensive than doing the rewrite in a way that Transifex more cleanly supports. It should
+also be mentioned that GitHub discourages the use of bots and separate bot accounts; they
+strongly recommend using GitHub Apps.
+
+Making a Transifex Project for each Repository
+==============================================
+
+As translation support is provided for more repos, the total cost will also increase. A
```

Comment:
I read "cost" as monetary, so the next two sentences were jarring. I might suggest:

```suggestion
As translation support is provided for more repos, the effort to maintain the translations infrastructure increases. A
```